opentelemetry-csharp-sdk-to-seq/Example.WeatherService/Program.cs

```diff
@@ -3,74 +3,60 @@
 using OpenTelemetry.Logs;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
+// ReSharper disable ExplicitCallerInfoArgument
 
-ActivitySource ExampleWeatherServiceActivitySource = new("Example Weather Service");
+var exampleActivitySource = new ActivitySource("Example.WeatherService");
 
-try
-{
-    var builder = WebApplication.CreateBuilder(args);
-
-    builder.Services.AddOpenTelemetry()
-        .ConfigureResource(resource => resource.AddService(serviceName: builder.Environment.ApplicationName))
-        .WithTracing(tracing =>
-        {
-            tracing.AddSource("Example Weather Service");
-            tracing.AddAspNetCoreInstrumentation();
-            tracing.AddConsoleExporter();
-            tracing.AddOtlpExporter(opt =>
-            {
-                opt.Endpoint = new Uri("http://localhost:5341/ingest/otlp/v1/traces");
-                opt.Protocol = OtlpExportProtocol.HttpProtobuf;
-                //opt.Headers = "X-Seq-ApiKey=abcde12345";
-            });
-        });
-
-    builder.Services.AddLogging(logging => logging.AddOpenTelemetry(openTelemetryLoggerOptions =>  
-    {  
-        openTelemetryLoggerOptions.SetResourceBuilder(  
-            ResourceBuilder.CreateEmpty()
-                .AddService(serviceName: builder.Environment.ApplicationName)  
-                .AddAttributes(new Dictionary<string, object>  
-                {
-                    // Add any desired resource attributes here
-                    ["deployment.environment"] = "development"  
-                }));
-  
-        // Some important options to improve data quality
-        openTelemetryLoggerOptions.IncludeScopes = true;
-        openTelemetryLoggerOptions.IncludeFormattedMessage = true;
+var builder = WebApplication.CreateBuilder(args);
 
-        openTelemetryLoggerOptions.AddConsoleExporter();
-        openTelemetryLoggerOptions.AddOtlpExporter(opt =>
+builder.Services.AddOpenTelemetry()
+    .ConfigureResource(resource => resource.AddService(serviceName: builder.Environment.ApplicationName))
+    .WithTracing(tracing =>
+    {
+        tracing.AddSource(exampleActivitySource.Name);
+        tracing.AddAspNetCoreInstrumentation();
+        tracing.AddConsoleExporter();
+        tracing.AddOtlpExporter(opt =>
         {
-            opt.Endpoint = new Uri("http://localhost:5341/ingest/otlp/v1/logs");
+            opt.Endpoint = new Uri("http://localhost:5341/ingest/otlp/v1/traces");
             opt.Protocol = OtlpExportProtocol.HttpProtobuf;
             //opt.Headers = "X-Seq-ApiKey=abcde12345";
         });
-    }));
+    });
 
-    var app = builder.Build();
-    app.Logger.LogInformation("Starting {App}", builder.Environment.ApplicationName);
+builder.Services.AddLogging(logging => logging.AddOpenTelemetry(openTelemetryLoggerOptions =>  
+{  
+    openTelemetryLoggerOptions.SetResourceBuilder(  
+        ResourceBuilder.CreateEmpty()
+            .AddService(serviceName: builder.Environment.ApplicationName));
```

Comment:
Removed the example resource attributes; these are normally set through environment variables and other means (although adding them here is still valid).